dipy/reconst/wigner.py

```diff
@@ -0,0 +1,618 @@
+import os
+import numpy as np
+import collections
+from scipy.linalg import block_diag
+from functools import lru_cache
+from dipy.data import get_fnames
+
+fetch_Jmat_path = get_fnames('wigner_jmat')
+print('fetched ' + fetch_Jmat_path)
+Jd = np.load(fetch_Jmat_path, allow_pickle=True)
```

Comment:
- can you remove the print ?
- it should not be global. please, create a decorator or just a function

Reply:
created a new function